mne/viz/_3d.py

```diff
@@ -581,16 +582,13 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
         It corresponds to Freesurfer environment variable SUBJECTS_DIR.
     surfaces : str | list
         Surfaces to plot. Supported values:
-
```

Comment:
@GuillaumeFavelier can you avoid such cosmetic changes? it makes diff longer to read for little gain. thx

Reply:
Yes of course. my bad, I'll be more careful.